### PR TITLE
select added

### LINKED ISSUE #4: Create Select component
The select component should accept an array of options as props.
In the similar format:
const options = [ {value: "elo", label: "elo"} ];
The component will be initialized like:
<Select value={value} onChange={(event) => setValue(event.target.value)} options={options} />
Inside the component, select should map over the array of options and render them on the screen.

### PR DESCRIPTION
**Pull request by Kszalank**
Closes #4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,9 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.8.1",
-    "sass": "^1.58.0"
+    "react-uuid": "^2.0.0",
```
Comment (mecies) [inline package.json]: Remove the library 
`npm uninstall react-uuid`
```suggestion
```

File: src/components/Selector.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import uuid from "react-uuid";
```
Comment (mecies) [inline src/components/Selector.jsx]:
```suggestion

```

File: src/components/Selector.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+function Select({ value, onChange, options }) {
+  return (
+    <select onChange={onChange} value={value}>
+      {options.map((element, index) => (
+        <option key={uuid()} label={options[index].label}>
+          {options[index].value}
+        </option>
+      ))}
+    </select>
```
Comment (mecies) [inline src/components/Selector.jsx]: Use the element directly instead of relying on index in `.map` 😄 + added restructuring to map
```suggestion
    <select onChange={onChange} value={value}>
      {options.map((({ value, label })) => (
        <option key={value} label={label}>
          {value}
        </option>
      ))}
    </select>
```

**Review by mecies — CHANGES_REQUESTED** [review]
Pay attention to details 💅 
The code works which is already good but you are already better than writing just "working" code 😄

File: package.json (hunk 1 above)
Comment (Kszalank) [reply package.json]: done

File: src/components/Selector.jsx (hunk 2 above)
Comment (Kszalank) [reply src/components/Selector.jsx]: done

File: src/components/Selector.jsx (hunk 3 above)
Comment (Kszalank) [reply src/components/Selector.jsx]: done, I had to change "function Select({ selectedValue, onChange, options })" from value to selectedValue, because it was declared twice

File: src/pages/CurrenciesListPage.jsx [hunk 4]
```diff
@@ -1,11 +1,25 @@
...
+    { value: "elo", label: "elo" },
+    { value: "siema", label: "siema" },
+  ];
+  const [value, setValue] = useState("");
   return (
     <div>
       <Heading variant="title">Currency Converter</Heading>
       <Heading variant="subtitle">Choose base currency</Heading>
+      <Select
+        selectedValue={value}
```
Comment (mecies) [inline src/pages/CurrenciesListPage.jsx]: It's a bad practice to rename the default HTML attribute names when providing props for them, please revert the change and rename the `useState` instead
```ts
const [selectedValue, setSelectedValue] = useState("");
```

Comment (Kszalank) [reply src/pages/CurrenciesListPage.jsx]: done

**Review by mecies — APPROVED** [review]
